docs/src/digitising-process.md

```diff
@@ -1,184 +1,349 @@
+   The Central Meridian (lon_0) and the Latitude of Origin (lat_0) were taken from the Zimbabwe sheets and the **+a**
+   and **+rf** values are for the Arc 1950 datum.
+
+#### Georefencing Parameters for the four sheets
+Georeferencing is an iterative process, and we needed to try different parameters to establish the most suitable ones 
+to use.
+
+The methods below describe the different parameters that were tested with the raster images.
+
+##### Test Parameters
 
-### Georeferencing Zimbabwe Sheets 1 through 4
+1. The first iteration of georeferencing Zimbabwe Sheet 1 was done using a `Thin Plate Spline transformation` using 11
+   GCPS. There were multiple discrepancies between the georeferenced image and the reference image.
 
-1. The first iteration of georeferencing Zimbabwe Sheet 1 was done using a Thin Plate Spline transformation using 11 GCPS. There were multiple discrepancies between the georeferenced image and the reference image.
+2. The first iteration of georeferencing Zimbabwe Sheet 2 was done using a `Thin Plate Spline transformation` using 12
+   GCPS. There were multiple discrepancies between the georeferenced image and the reference image.
 
-2. The first iteration of georeferencing Zimbabwe Sheet 2 was done using a Thin Plate Spline transformation using 12 GCPS. There were multiple discrepancies between the georeferenced image and the reference image.
+3. The first iteration of georeferencing Zimbabwe Sheet 3 was done using a `Thin Plate Spline transformation` using 8
+   GCPS. There were multiple discrepancies between the georeferenced image and the reference image.
 
-3. The first iteration of georeferencing Zimbabwe Sheet 3 was done using a Thin Plate Spline transformation using 8 GCPS. There were multiple discrepancies between the georeferenced image and the reference image.
+##### Best Candidate Parameters to Use
 
-4. The first iteration of georeferencing Zimbabwe Sheet 4 was done using a Thin Plate Spline transformation using 12 GCPS. There were multiple discrepancies between the georeferenced image and the reference image. It was the best result of the initial georeferencing attempts so will be refined first and then used as a basis for the other images.
+The first iteration of georeferencing Zimbabwe Sheet 4 was done using a `Thin Plate Spline transformation` using 12
+GCPS. There were multiple discrepancies between the georeferenced image and the reference image. It was the best
+result of the initial georeferencing attempts so will be refined first and then used as a basis for the other images.
 
 ### Refining Zimbabwe Sheet 4
 
-1. The initial refinement of Zimbabwe sheet 4 was done using a Thin Plate Spline transformation with 46 GCPS. There were still discrepancies between the GAUL reference dataset and the georeferenced sheet's boundaries.
+1. The initial refinement of Zimbabwe sheet 4 was done using a `Thin Plate Spline transformation` with 46 GCPS. There were
```

Comment:
Can we also not do a table and have an explanation below the table ?

Reply:
I have made tables for all of the refining steps

Reply:
Awesome, Are you doing it in your branch and will do a PR afterwards

Reply:
I'm doing it on my VS code copy of the repo and will wait to sync until the other PR has gone through otherwise it will all merge into one PR